### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by jurekbarth**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,10 +12,10 @@
     "build": "tsc"
   },
   "dependencies": {
-    "@biotope/generator": "^2.6.0",
+    "@biotope/generator": "^2.7.0",
```
Comment (SheepFromHeaven) [inline package.json]: let's just do ^2

**Review by SheepFromHeaven — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -12,10 +12,10 @@
     "build": "tsc"
   },
   "dependencies": {
-    "@biotope/generator": "^2.6.0",
+    "@biotope/generator": "^2",
     "commander": "^2.15.1",
     "git-clone": "^0.1.0",
-    "npm": "^6.4.1",
+    "npm": "^6.9.0",
```
Comment (tiagomapmarques) [inline package.json:18]: Q: Do we need this dep? Is it not enough to have `engines` in `package.json`?

Comment (jurekbarth) [reply package.json:18]: Mhmm good question :) I don't know let's try remove it and see what happens 👍

**Comment by SheepFromHeaven** [comment]
@jurekbarth after adding the dependency bot, is this still a topic?

**Comment by timomayer** [comment]
in my opinion this PR makes sense even with depfu bot. but i think Tiagos question is a valid one and should be answered or npm removed before merging.

File: package.json (hunk 2 above)
Comment (SheepFromHeaven) [reply package.json:18]: @tiagomapmarques @jurekbarth this is a dependency, as we run npm by node and not by executing the cli

**Review by timomayer — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (timomayer) [reply package.json:18]: @biotope/core-enforcer second approval needed for merge

**Review by timomayer — APPROVED** [review]

**Review by SheepFromHeaven — APPROVED** [review]

**Comment by SheepFromHeaven** [comment]
@biotope/core-enforcer this one was already merged bit by bit and is now stale. Will close it

**Pull request closed without merging**